WordPress/src/test/java/org/wordpress/android/ui/bloggingprompts/editor/EditorBloggingPromptsViewModelTest.kt

```diff
@@ -99,4 +99,21 @@ class EditorBloggingPromptsViewModelTest : BaseUnitTest() {
         viewModel.start(siteModel, 123)
         assertThat(loadedPrompt?.content).isEqualTo(bloggingPromptsBlock)
     }
+
+    @Test
+    fun `should not add specific prompt tag if enhancements feature flag is DISABLED`() {
+        whenever(bloggingPromptsEnhancementsFeatureConfig.isEnabled()).thenReturn(false)
+        viewModel.start(siteModel, 123)
+        assertThat(loadedPrompt?.tags).containsOnly(BLOGGING_PROMPT_TAG)
+    }
+
+    @Test
+    fun `should add specific prompt tag if enhancements feature flag is ENABLED`() {
```

Comment:
Nitpick: if it makes sense to rename "specific tag" to "prompt ID tag", would it make sense to rename this test?

e.g.
`should add prompt ID tag if enhancements feature flag is ENABLED`